autometa/common/external/samtools.py

```diff
@@ -100,5 +84,4 @@ def main(args):
     parser.add_argument('--bam', help='</path/to/output/alignment.bam>', type=str)
     parser.add_argument('--nproc', help='Number of processors to use', default=mp.cpu_count(), type=int)
     args = parser.parse_args()
-    main(args)
-
+    main(args)
```

Comment:
It appears GitHub is telling us there should be a newline at the end of the file.

Reply:
I added a new line. Although is there any specific reason to do it? I mean what would happen if we don't. Just curious.

Reply:
See discussion here: https://stackoverflow.com/a/5813359